index.html

```diff
@@ -108,22 +123,23 @@ <h3>I'm a software developer</h3>
         </ul>
       </div>
     </section>
-    <section class="works" id = "portfolio">
+    <section class="works" id="portfolio">
       <div class="recent-works">
-        <div class="title"> 
+        <div class="title">
           <div class="title-h1">
             <h1>My Recent Works</h1>
           </div>
           <div class="line"></div>
         </div>
         <div class="section-2">
-          <div> <img src="img/Placeholder.svg" alt="" /> </div>
-          <div class="multi-post-stories"> 
-             <h2>Multi-Post Stories</h2>
+          <div><img src="img/Placeholder.svg" alt="" /></div>
+          <div class="multi-post-stories">
+            <h2>Multi-Post Stories</h2>
```

Comment:
- [ ] Hi @sangwa7, you haven't still resolved this issue of the pop-up model functionalities pointed out by the previous reviewers for the first project. When the user clicks the **see project** button to check project details, the popup with details about the project should appear. Please kindly fix this issue and submit your project once more.